boards/avr-rss2/Makefile.features

```diff
@@ -0,0 +1,19 @@
+
+CPU = atmega256rfr2
+
+# This board is based on an atmega CPU, thus import the features from it
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.features
+
+# Put defined MCU peripherals here (in alphabetical order)
+# Peripherals are defined in common/arduino-atmega/Makefile.features
+# Add only additional Peripherals
+
+# Various other features (if any)
+FEATURES_PROVIDED += arduino
+FEATURES_PROVIDED += periph_adc
```

Comment:
The peripheral features are not "various other features". Only arduino is. So all features provided that are related to peripherals should move to the paragraph above.
```mk
# Put defined MCU peripherals here (in alphabetical order)
FEATURES_PROVIDED += periph_adc
...

# Various other features (if any)
FEATURES_PROVIDED += arduino

```